cmake/Config.cmake

```diff
@@ -209,6 +209,10 @@ set (BOND_ENABLE_COMM
     "FALSE"
     CACHE BOOL "If FALSE, then do not build Comm")
 
+set (BOND_ENABLE_JAVA
```

Comment:
The `GRADLE_FOUND` and `MAVEN_FOUND` variables in FindJavaBuildTools.cmake don't appear to be used. Can they be removed?

Reply:
Done.